template/app/src/main/java/com/xmartlabs/template/module/ControllerModule.kt

```diff
@@ -0,0 +1,26 @@
+package com.xmartlabs.template.module
+
+import com.xmartlabs.bigbang.core.controller.CoreSessionController
+import com.xmartlabs.template.controller.AuthController
+import com.xmartlabs.template.controller.SessionController
+import com.xmartlabs.template.model.Session
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+@Module
+class ControllerModule {
+  @Provides
+  @Singleton
+  internal fun provideAuthController() = AuthController()
+
+  @Provides
+  @Singleton
+  internal fun provideCoreSessionController(sessionController: SessionController): CoreSessionController {
+    return sessionController
```

Comment:
Right now now, but with #123 this class wont be needed anymore (i think). With a couple of extensions we should be ok